clang/lib/Driver/Driver.cpp

```diff
@@ -2945,6 +2966,12 @@ class OffloadingActionBuilder final {
     /// The OpenMP actions for the current input.
     ActionList OpenMPDeviceActions;
 
+    bool CompileHostOnly = false;
```

Comment:
This device-only openmp compilation is likely to be needed to build the deviceRTL from openmp, instead of from cuda + hip.